src/scripts/main.js

```diff
@@ -354,7 +354,28 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashBoard = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((person) => {
+  const trElem = document.createElement('tr');
+  const tdName = document.createElement('td');
+  const tdGender = document.createElement('td');
+  const tdBorn = document.createElement('td');
+  const tdDied = document.createElement('td');
+  const tdAge = document.createElement('td');
+  const tdCentury = document.createElement('td');
+
+  dashBoard.append(trElem);
+  trElem.append(tdName);
+  trElem.append(tdGender);
+  trElem.append(tdBorn);
+  trElem.append(tdDied);
+  trElem.append(tdAge);
+  trElem.append(tdCentury);
```

Comment:
You have a lot of duplicates, will be better if you create array with table data and use method for simplify code

Reply:
array with table data was created by MA in the task, and I use ForEach so I don't understand what do you want I have to do.

Reply:
also I think it's not simplify my code, becouse i have to create array with 6 objects, td, and in function use 6 time if for apply different content for each one, this isn't simplify, this is more complicated

Reply:
i have been thinking about your review and found best option. I think now my code more simplify and don't have a lot of duplicates